assets/javascripts/discourse/widgets/discourse-reactions-state-panel.js.es6

```diff
@@ -55,20 +55,23 @@ export default createWidget("discourse-reactions-state-panel", {
       return;
     }
 
+    const divContainer = attrs.state.postIds.includes(attrs.post.id) ? h(
+      "div.counters",
+      attrs.post.reactions.map(reaction =>
+        this.attach("discourse-reactions-state-panel-reaction", {
+          reaction,
+          users: attrs.state[reaction.id],
+          post: attrs.post,
+          isDisplayed: reaction.id === this.state.displayedReactionId
+        })
+      )
+    ) : h('div.center', h('div.spinner'));
```

Comment:
I don't like this center, it describes the expected behavior, not what this object is about.

html should be decorelated from css, what if you to not have it centered in another theme ?

Reply:
that's true will change it